cmake/modules/hunter_autotools_project.cmake

```diff
@@ -98,7 +98,7 @@ function(hunter_autotools_project target_name)
 
   if(PARAM_UNPARSED_ARGUMENTS)
     hunter_internal_error(
-        "Invalid arguments passed to hunter_autotools_configure:"
+        "Invalid arguments passed to hunter_autotools_project:"
```

Comment:
This is the only requirement for now, if you update INTERNAL_ID I will run tests and merge this pull request.